features/bicycle/ref.feature

```diff
@@ -25,8 +25,8 @@ Feature: Bike - Way ref
             | ab    |      | E7  |
 
         When I route I should get
-            | from | to | route |
-            | a    | b  | E7,E7 |
+            | from | to | route                               | ref   |
+            | a    | b  | {highway:primary},{highway:primary} | E7,E7 |
```

Comment:
Looking at the OSM Wiki for ref:
http://wiki.openstreetmap.org/wiki/Key:ref

it might make sense now to also try more namespaced refs, as in: try the following tags in order:
-  `ref=`
- `ncn_ref=`
- `rcn_ref=`
- `lcn_ref=`

for the bike profile.

A similar approach can be taken for the car profile I guess.